cfpb/consumerfinance.gov
# Speed up initial render and hydration of pfc grad path

Closes https://github.com/cfpb/consumerfinance.gov/issues/8219

This PR does two things:
  - Adds and utilizes some bulk updating methods to prevent unnecessary recalculations (primarily in the financial model). Recalculations in the models only happen after all the data has been seeded from the params.
  - Checks the radio buttons on the landing page directly instead of emitting clicks (which then trigger a bunch of unneeded/unnecessary events and processing logic)/

## Testing
 - Visit [prod](https://www.consumerfinance.gov/paying-for-college/your-financial-path-to-graduation/?iped=163286&pid=4502-3&houp=onCampus&typp=bachelors&prop=0&lenp=3&ratp=inState&depp=dependent&cobs=n&tuit=10595&hous=12809&diro=13&book=1250&indo=55&tran=44&pelg=11&seog=12&fedg=13&stag=14&schg=15&tuig=16&othg=17&mta=18&gi=19&othm=20&stas=21&schs=22&oths=23&wkst=33&subl=299&unsl=399&insl=599&insr=0.06&insf=0.02&stal=499&star=0.05&staf=0.01&npol=699&npor=0.07&npof=0.03&pers=81&fams=82&529p=83&offj=84&onj=85&eta=86&othf=87&houx=31&fdx=32&clhx=33&trnx=34&hltx=35&entx=36&retx=37&taxx=38&chcx=39&othx=41&dbtx=40) in one window. It will be very slow to both initially load and then to populate the radio buttons (5-15s for each)
 - Pull, build, fire up localhost and visit [localhost:8000](http://localhost:8000/paying-for-college/your-financial-path-to-graduation/?iped=163286&pid=4502-3&houp=onCampus&typp=bachelors&prop=0&lenp=3&ratp=inState&depp=dependent&cobs=n&tuit=10595&hous=12809&diro=13&book=1250&indo=55&tran=44&pelg=11&seog=12&fedg=13&stag=14&schg=15&tuig=16&othg=17&mta=18&gi=19&othm=20&stas=21&schs=22&oths=23&wkst=33&subl=299&unsl=399&insl=599&insr=0.06&insf=0.02&stal=499&star=0.05&staf=0.01&npol=699&npor=0.07&npof=0.03&pers=81&fams=82&529p=83&offj=84&onj=85&eta=86&othf=87&houx=31&fdx=32&clhx=33&trnx=34&hltx=35&entx=36&retx=37&taxx=38&chcx=39&othx=41&dbtx=40). Loading should be much faster, as should populating the radio buttons. (1-2s)
